<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuItemWidget</class>
 <widget class="QWidget" name="MenuItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>افزودن به سبد خرید</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="priceLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="quantitySpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
